### DISCUSSION TITLE
Is there a standalone version or inspiration package of the forms used in Inertia-js?

### DISCUSSION THREAD

**Discussion by sneakylenny** [inertiajs/inertia#908] [Help] [answered]
I'm currently working on a Nuxt.js project that consumes a Laravel API. The thing is: I'm so used to Inertia's forms I was wondering if there is a package that works similar of if I can even import Inertia just to just to use that feature. Does anyone know?

**Comment by sneakylenny** [comment]


**Reply by johannesronaldsson** [reply]
Any updates on this? Would be great to make it a public library 👯

**Reply by sneakylenny** [reply]
I put some effort in turning it into a customizable package!
~https://www.npmjs.com/package/nuxt-form-helper~ - **deleted**
~https://github.com/timyourivh/NuxtFormHelper~ - **deleted**
I hope it helps, it did for me. Let me know if you find any bugs or missing features!

**EDIT:**
I deleted it due to no one having any interest in it.

**Comment by sneakylenny** [comment, marked as answer]
I've made my own:
~https://www.npmjs.com/package/nuxt-form-helper~ - **deleted**
~https://github.com/timyourivh/NuxtFormHelper~ - **deleted**

**EDIT:**
Due to not needing it myself anymore and Nuxt 3 being out, I've decided to delete the package since no one has or will have any interest in it. I may create a helper for Nuxt 3 in the future, but **if** I do, I will start from scratch anyway with Inertia as a strong inspiration.

**EDIT2:**
For those who use Laravel and still would like to use something similar, Laravel now has it's own "js-to-laravel-form-helpers" under the name [Laravel Precognition](https://laravel.com/docs/10.x/precognition)
